configs/projects/client-tools-runtime-2021.7.x.rb

```diff
@@ -1,4 +1,6 @@
 project 'client-tools-runtime-2021.7.x' do |proj|
+  proj.setting :ruby_version, '2.7.8'
```

Comment:
Hello @joshcooper ,
Don't we want to add this just for macos only?

Reply:
sorry forgot to push recent changes